app/Sidebar/Sidebar.php

```diff
@@ -124,12 +184,30 @@ public function addAcfFields(): void
             'position' => 'normal',
             'style' => 'default',
             'label_placement' => 'top',
-            'instruction_placement' => 'label',
+            'instruction_placement' => 'field',
             'hide_on_screen' => '',
             'active' => true,
             'description' => '',
             'show_in_rest' => 0,
-            'modified' => 1727360643,
         ]);
     }
+
+    public function getMunicipalityIcons(): array
+    {
+        $default = ['none' => 'Geen icoon geselecteerd'];
+        $directory = get_template_directory() . '/assets/img/municipality-icons/*.svg';
```

Comment:
Staan de icons altijd in het hoofdthema? Of kunnen ze in het subthema overschreven worden?